src/benchmark/operators/join_benchmark.cpp

```diff
@@ -34,15 +37,15 @@ void clear_cache() {
 }  // namespace
 
 namespace hyrise {
-
+namespace {
```

Comment:
On purpose?

Reply:
Yes.

I am not sure what to think about nesting them. But we usually put them on top of the hyrise namespace. Maybe something to discuss on Wednesday.

Reply:
What exactly do you mean by "on top"? Before the hyrise namespace or at the top, but within the hyrise namespace?

Reply:
Outside the the hyrise namespace. We usually have it like that:
```
#include ...

namespace {
using hyrise;
} // namesapce

namespace hyrise {
} // namespace
```

It's probably the exactly the same in the end. But I'd prefer to have it consistent.